devicetypes/smartthings/zigbee-button.src/zigbee-button.groovy

```diff
@@ -215,6 +215,7 @@ def configure() {
 private Map getButtonResult(buttonState, buttonNumber = 1) {
     if (buttonState == 'release') {
         log.debug "Button was value : $buttonState"
+        state.pressTime = now()
```

Comment:
Making this change means the button will always be `pushed` and never `held`

Reply:
yeah I think you just want to add a null check for `state.pressTime`

Reply:
@tpmanley button has 3 statuses. I made this change because when device is pairing there was an error about null value. 
should I create null check(in installed) or leave it in this method?

Reply:
Think about what the original code is intended to do. 

`state.pressTime` is set to `now()` when the button is pressed. When the button is released a separate event is sent. We subtract those two timestamps and then determine whether the button was pressed or held based on the delta. If you set `state.pressTime` to `now()` right before that subtraction every time, the delta will always be <1ms, so "held" will never be sent.

The error you're seeing is because a button release event was sent before `state.pressTime` was set. If `state.pressTime` is not set it makes the most sense to do nothing. So what you should be doing is:

* add a short circuit `[:]` return if `state.pressTime`is `null`
* verify that you are receiving the events you expect from the device.